tests/torchtune/data/test_data_utils.py

```diff
@@ -13,8 +13,9 @@ def test_tokenize_prompt_and_response():
     tokenizer = DummyTokenizer()
     prompt = "Instruction:\nThis is an instruction.\n\nInput:\nThis is an input.\n\nResponse: "
     response = "I always know what I'm doing, do you?"
-    prompt_length = 11
+    prompt_length = 12
     expected_tokenized_prompt = [
+        -1,
```

Comment:
Maybe I'm being dumb here, but why is the first token the dummy tokenizer's eos ID?

Reply:
huh why did this pass